templ/maven+service-proxy/pom.xml

```diff
@@ -15,17 +15,61 @@
   {{/if}}
 
   <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+
     <main.verticle>{{metadata.packageName}}.MainVerticle</main.verticle>
   </properties>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>io.vertx</groupId>
+        <artifactId>vertx-dependencies</artifactId>
+        <version>{{coreVersion}}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      {{#if hasUnit}}
+
+      <dependency>
+        <groupId>io.vertx</groupId>
+        <artifactId>vertx-unit</artifactId>
+        <version>{{coreVersion}}</version>
+        <scope>test</scope>
+      </dependency>
+
+      <dependency>
```

Comment:
if we add `vertx-unit` it should pull junit, since it is an optional dependency it should be available and we don't exclude dependencies anywhere in the simple pom's